test/Prototypes/Integers.swift.gyb

```diff
@@ -45,21 +45,20 @@ class struct(object):
   def __repr__(self):
     return 'struct(%r)' % self.__dict__
 
-binaryArithmetic = [
-       struct(operator='+', name='adding',      mutatingName='add',           firstArg='',           llvmName='add', kind='+'),
-       struct(operator='-', name='subtracting', mutatingName='subtract',      firstArg='',           llvmName='sub', kind='-'),
-       struct(operator='*', name='multiplied',  mutatingName='multiply',      firstArg='by',         llvmName='mul', kind='*'),
-       struct(operator='/', name='divided',     mutatingName='divide',        firstArg='by',         llvmName='div', kind='/'),
-       struct(operator='%', name='remainder',   mutatingName='formRemainder', firstArg='dividingBy', llvmName='rem', kind='/')
+  binaryArithmetic = [
```

Comment:
@practicalswift  I don't know why your local test passes.
This change modifies the scope of `binaryArithmetic`.

```python
class Foo(object):
    def __init__(self):
        pass

    foo = Foo()
```
```
Traceback (most recent call last):
  File "test.py", line 2, in <module>
    class Foo(object):
  File "test.py", line 6, in Foo
    foo = Foo()
NameError: name 'Foo' is not defined
```